### DISCUSSION TITLE
Better window snapping

### DISCUSSION THREAD

**Discussion by erikjanp** [OpenRCT2/OpenRCT2#16431] [Feature requests]
In OpenRCT2, windows snap when they are close to each other, but only when the left/top side of one window is close to the right/bottom side of the other window. 
There is no snapping to the same side (left/left, top/top etc) so you can aligning your windows nicely is quite tricky.
Here's a patch that expands window snapping so lining up your windows neatly is much easier.

https://forums.openrct2.org/applications/core/interface/file/attachment.php?id=8960&key=4891ee87bac4b4c9ba0b59da78fb0d8c
Patch is against v0.3.5.1

**Comment by janisozaur** [comment]
Please submit a PR instead
